README.md

```diff
@@ -123,6 +124,9 @@ this.state = {
         justifyContent: 'flex-start'
     }}
     onChangeItem={item => this.setState({
+        countryValues: item // an array of the selected items values
+    })}
+    onChangeItemMultiple={item => this.setState({
         countries: item // an array of the selected items
```

Comment:
Here when the user selects an item, the whole items will be reset and the selected item is the only available item in the dropdown box.

Lets say there are 10 items, after selecting one of them, `onChangeItemMultiple` will be called and all of the available items are gone except the selected item (object).

Please provide a better example for `onChangeItemMultiple`